tests/oserror_aliases_test.py

```diff
@@ -8,6 +8,26 @@
 from pyupgrade import FindPy3Plus
 
 
+@pytest.mark.parametrize('imp', FindPy3Plus.OS_ERROR_ALIAS_MODULES)
+@pytest.mark.parametrize(
+    'tpl',
+    (
+        # if "error" isn't a function call it shouldn't be rewritten
```

Comment:
I think we should handle `except` blocks -- they're far more likely than construction of these error classes imo

Reply:
More generally, should I change the OSError fixer to only change `raise` and `except` blocks?

Reply:
yeah probably

Reply:
done